daemon/algod/api/Makefile

```diff
@@ -23,6 +23,6 @@ algod.oas3.yml:	.3tmp.json
 	curl -s -X POST "https://converter.swagger.io/api/convert" -H "accept: application/json" -H "Content-Type: application/json" -d @./algod.oas2.json -o .3tmp.json
 
 oapi-codegen:	.PHONY
-	GO111MODULE=on go get -u "github.com/algorand/oapi-codegen/...@v1.3.5-algorand4"
+	GO111MODULE=on go get -u "github.com/algorand/oapi-codegen/...@v1.3.5-algorand5"
```

Comment:
we need to find a way to derive that from the go.mod, so that upgrading the oapi-codegen would have a single place where we would make the change. but that's not an issue for this PR.

Reply:
I'm open to suggestions for how to do this. Technically it's not a bug if they mismatch because it's a library and a cli tool. So far changes have been to one or the other, not both. It doesn't look right to have different versions though, so I've been trying to keep them synchronized.

Reply:
This gets me pretty close?
`grep oapi-codegen go.mod | awk '{ print $1 "/...@" $2 }'`